react-backend/routes/__tests__/qr.generator.test.js

```diff
@@ -0,0 +1,64 @@
+const test = require('tape');
+const request = require('supertest');
+const jwt = require('jsonwebtoken');
+const createApp = require('../../app');
+const { getConfig } = require('../../../config');
+
+const config = getConfig(process.env.NODE_ENV);
+
+test('POST /api//qr/generator | Correct payload', t => {
```

Comment:
Double `//` and in all the calls to `supertest`. Does express ignore the double slash? How are the tests working? 😕